### PR TITLE
Calculate

### PR DESCRIPTION
**Pull request by Papierkowy**

### CODE REVIEW

File: module1/homework/calculate/calculate.hpp [hunk 1]
```diff
@@ -3,5 +3,19 @@
 
 std::string calculate(const std::string& command, int first, int second) {
     // TODO: Implement your solution here and return proper value
-    return "";
+    if (command == "add"){
+        return std::to_string(first + second);
+    } else
```
Comment (Pawellos) [inline module1/homework/calculate/calculate.hpp:8]: Why else here ?

File: module1/homework/calculate/calculate.hpp [hunk 2]
```diff
@@ -3,5 +3,19 @@
...
+     if (command == "subtract"){
+        return std::to_string(first - second);
+    } 
+     if (command == "multiply"){
+        return std::to_string(first * second);
```
Comment (kuchichan) [inline module1/homework/calculate/calculate.hpp:13]: Please use auto-formatter :)

File: module1/homework/calculate/calculate.hpp [hunk 3]
```diff
@@ -3,5 +3,19 @@
...
+    }  if (command == "divide"){
+        if(second == 0){
+             return "Division by 0";;
```
Comment (Vicx95) [inline module1/homework/calculate/calculate.hpp:16]: you have two semicolons here
```suggestion
             return "Division by 0";
```

File: module1/homework/calculate/calculate.hpp [hunk 4]
```diff
@@ -3,5 +3,19 @@
 
 std::string calculate(const std::string& command, int first, int second) {
     // TODO: Implement your solution here and return proper value
-    return "";
+    if (command == "add"){
+        return std::to_string(first + second);
+    } else
+     if (command == "subtract"){
```
Comment (nauka-programowania-MA) [inline module1/homework/calculate/calculate.hpp:9]: Write `else if` in one line :)

File: module1/homework/calculate/calculate.hpp [hunk 5]
```diff
@@ -3,5 +3,19 @@
...
+        return std::to_string(first - second);
+    } 
+     if (command == "multiply"){
```
Comment (nauka-programowania-MA) [inline module1/homework/calculate/calculate.hpp:12]: Use here also `else if`

File: module1/homework/calculate/calculate.hpp [hunk 6]
```diff
@@ -3,5 +3,19 @@
...
+        return std::to_string(first * second);
+    }  if (command == "divide"){
```
Comment (nauka-programowania-MA) [inline module1/homework/calculate/calculate.hpp:14]: and `else if` here

**Comment by nauka-programowania-MA** [comment]
{
"requirement_ids": [1429],
"points": [5]
}

**Pull request closed without merging**